### ISSUE TITLE
I need help 

### ISSUE THREAD

**Issue by dr1408** [Justxd22/Eviltwin-Huawei_XD#3]
Hello salam alikum thanks for this great project im facing only one problem and i dunno why .. its related to passapi.py .. im trying to test this locally ..

the handshake should he in main folder or inside the portal folder ? 

![A98CE421-E501-464C-84FD-81C97359F455](https://github.com/user-attachments/assets/51e28ba2-90f8-4528-8dcc-878a46e72d3c)
![A4414507-45B0-427D-A7DF-56FF1703A9D4](https://github.com/user-attachments/assets/c2250923-5995-4a96-ac9e-9581ee12765b)

**Comment by dr1408** [comment]
Ok brother i made it work succesfully on nethunter .. just wanna ask if you have another portals that work with same method .. thanks alot again 🙏🏼

**Comment by Justxd22** [comment]
@dr1408 glad you liked the project :D 

pcap file same path as passapi.py

Don't visit 0.0.0.0:5000 that's for the api to work! the portal can be hosted using any webserver like apache or nignix or python simple http then edit the portal and point it to api url

That's the only portal style i made for Huawei as my target was using Huawei router, BUT you can still find a lot of portals in
 https://github.com/FluxionNetwork/fluxion/tree/master/attacks%2FCaptive%20Portal%2Fsites

**Comment by dr1408** [comment]
Liked it alot actually i did script for running speacial iptables and ip rules for android .. internet source is either wlan0 or mobile internet .. do you have telegram ? And how i can add the script that verify handshale to this portals pls

**Comment by dr1408** [comment]
Handshake sorry

**Comment by dr1408** [comment]
Im thinking to edit the frameware upgrade one from wifiphisher tool

**Comment by Justxd22** [comment]
@dr1408 yes i do have telegram! Use my website https://justxd22.github.io

**Comment by Justxd22** [comment]
@dr1408 your hotspot should already have internet source as android handles everything for you, The only problem is you need 200.200.200.x ip dhcp so when devices are connected portal is triggered
